app/src/main/java/io/mochahub/powermeter/exercises/NewExerciseDialog.kt

```diff
@@ -0,0 +1,69 @@
+package io.mochahub.powermeter.exercises
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import androidx.fragment.app.DialogFragment
+import android.view.ViewGroup
+import androidx.lifecycle.ViewModelProviders
+import io.mochahub.powermeter.R
+import io.mochahub.powermeter.models.Exercise
+import kotlinx.android.synthetic.main.dialog_new_exercise.*
+
+class NewExerciseDialog : DialogFragment() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setStyle(STYLE_NORMAL, R.style.FullScreenDialog)
+    }
+
+    override fun onStart() {
+        super.onStart()
+
+        if (dialog != null) {
+            val width = ViewGroup.LayoutParams.MATCH_PARENT
+            val height = ViewGroup.LayoutParams.MATCH_PARENT
+            dialog!!.window!!.setLayout(width, height)
+        }
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        super.onCreateView(inflater, container, savedInstanceState)
+        // TODO: The toolbar's X icon isn't final, need to figure out how to make it custom to night mode
```

Comment:
Don't think it needs to be custom to night mode. It should always stay white I think, like the toolbar title.